### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by mads256h**
Connect to sqlite db in memory
Insert and get devices
Add tests
Use this in the WebUI

### CODE REVIEW

**Review by Taoshix — APPROVED** [review]
I like how organized this is from the beginning

開始からこれが整頓されたのが好きだ

File: CentralHub.Api/Program.cs [hunk 1]
```diff
@@ -14,6 +33,12 @@
 {
     app.UseSwagger();
     app.UseSwaggerUI();
+
+    // Insert dummy devices
+    var devicesRepository = app.Services.GetRequiredService<IDevicesRepository>();
+    devicesRepository.AddDevice(new Device("Test Device 1", "AA:BB:CC:DD:EE:FF", DeviceType.WiFi));
```
Comment (miniJmo) [inline CentralHub.Api/Program.cs:39]: look at github actions (add await)

**Review by miniJmo — COMMENTED** [review]
change device -> tracker

File: CentralHub.Api/Program.cs [hunk 2]
```diff
@@ -14,6 +33,12 @@
...
+    devicesRepository.AddDevice(new Device("Test Device 2", "FF:EE:DD:CC:BB:AA", DeviceType.WiFi));
+    devicesRepository.AddDevice(new Device("Test Device 3", "00:11:22:33:44:55", DeviceType.Bluetooth));
```
Comment (miniJmo) [inline CentralHub.Api/Program.cs:41]: same as before add await

**Review by miniJmo — COMMENTED** [review]
Remove everything mentioning weatherforcast

**Comment by miniJmo** [comment]
but other than that lgtm :shipit:

**Review by miniJmo — APPROVED** [review]
lgtm :shipit:

**Pull request merged**